tutorial/markdown/nodejs/quickstart-typescript/quickstart-typescript.md

```diff
@@ -0,0 +1,483 @@
+---
+# frontmatter
+path: "/tutorial-quickstart-nodejs"
+# title and description do not need to be added to markdown, start with H2 (##)
+title: Using Typescript with Node.js & Express
```

Comment:
Typescript -> TypeScript
Also make it consistent everywhere in the tutorial

Reply:
Changed it to TypeScript